run.py

```diff
@@ -106,6 +114,26 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        issue_engine = create_engine('sqlite:///' + os.path.join(app.config['BASE_DIR'], 'issues.db'))
+        issue_session_maker = sessionmaker(autocommit=False,
+                                           autoflush=False,
+                                           bind=issue_engine)
+        issue_db = scoped_session(issue_session_maker)
+        # Take a backup if issues.db has data dump.
+        # TODO: write code for creating issues dump in the issues.db with updated schema
```

Comment:
I think @karlcow is saying to remove the # TODO comment, since this PR is about backup (and schema updates are in the air).